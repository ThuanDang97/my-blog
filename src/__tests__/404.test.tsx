// Utils
import { render } from '@utils/testUtils'

// Page
import NotFoundPage from '@pages/404'

describe('Page Not Found', () => {
  test('Page Not Found matches DOM Snapshot', () => {
    const { container } = render(<NotFoundPage />)

    expect(container).toMatchSnapshot()
  })
})
