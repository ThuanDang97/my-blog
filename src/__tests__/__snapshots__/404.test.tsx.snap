// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page Not Found Page Not Found matches DOM Snapshot 1`] = `
<div>
  <div
    class="MuiBox-root css-wg3j4w"
  >
    <h1>
      Page Not Found
    </h1>
    <p
      class="zoom-area"
    >
      <b>
        Oops!
      </b>
       Something is wrong.
    </p>
    <section
      class="error-container"
    >
      <span
        class="four"
      >
        <span
          class="screen-reader-text"
        >
          4
        </span>
      </span>
      <span
        class="zero"
      >
        <span
          class="screen-reader-text"
        >
          0
        </span>
      </span>
      <span
        class="four"
      >
        <span
          class="screen-reader-text"
        >
          4
        </span>
      </span>
    </section>
    <div
      class="link-container"
    >
      <a
        href="/"
      >
        Go HomePage
      </a>
    </div>
  </div>
</div>
`;
